python/ray/tests/test_object_spilling.py

```diff
@@ -373,5 +419,59 @@ def test_spill_deadlock(object_spilling_config, shutdown_only):
                 assert np.array_equal(sample, arr)
 
 
+@pytest.mark.skipif(
+    platform.system() == "Windows", reason="Failing on Windows.")
+def test_fusion_objects(tmp_path, shutdown_only):
+    # Limit our object store to 75 MiB of memory.
+    temp_folder = tmp_path / "spill"
+    temp_folder.mkdir()
+    min_spilling_size = 30 * 1024 * 1024
+    ray.init(
+        object_store_memory=75 * 1024 * 1024,
+        _system_config={
+            "max_io_workers": 4,
+            "automatic_object_spilling_enabled": True,
+            "object_store_full_max_retries": 4,
+            "object_store_full_initial_delay_ms": 100,
+            "object_spilling_config": json.dumps({
+                "type": "filesystem",
+                "params": {
+                    "directory_path": str(temp_folder)
+                }
+            }),
+            "min_spilling_size": min_spilling_size,
+        })
+    arr = np.random.rand(1024 * 1024)  # 8 MB data
+    replay_buffer = []
+
+    # Wait raylet for starting an IO worker.
+    time.sleep(1)
```

Comment:
Is this needed? We should try not to depend on timing.

Reply:
Oh this part was copied from `test_object_spilling_automatically`. I will delete sleep for both tests.

Reply:
Or we should probably auto calculate this value (like at maximum 60% of plasma store size or something)